setup.py

```diff
@@ -257,6 +257,7 @@ def run(self):
             "pillow>=6.2.0",
             "pyparsing>=2.0.3,!=2.0.4,!=2.1.2,!=2.1.6",
             "python-dateutil>=2.1",
+            "cython",
```

Comment:
I'm sorry, but I have to call you back on this. There have been various discussions on adding cython as a dependency in the past, an we've decided againt it for the time being.

Reply:
Sorry that was push to the wrong branch.